midi-to-part-mp3s

```diff
@@ -26,6 +27,34 @@ class Part:
     def mp3_filepath(self):
         return self.midi_filepath.replace(".midi", ".mp3")
 
+def check_format(file_path: str) -> str:
+  """If the format is midi, proceed. If the format is MusicXML, convert to midi.
+  
+  Arguments:
+      file_path {[str]} -- path of the file to convert
+  
+  Raises:
+      NameError: Raised if trying to convert a format not supported by the script
+  
+  Returns:
+      str -- midi file name (original or converted midi file)
+  """
+  if file_path.endswith('.mid') or file_path.endswith('.midi'):
+    return file_path
+  elif file_path.endswith('.mxl'):
+    converted_file_path = output_directory + '/temp.midi'
```

Comment:
Lines 45-52 are could all be summed up as conversion from musicXML to midi, so for modularity can I ask you to split apart the conversion code into a separate function, something like `convert_musicxml_to_midi()`?

Reply:
Sure, I will change that